esp-hal/src/gpio/interconnect.rs

```diff
@@ -98,6 +98,7 @@ impl gpio::OutputSignal {
     pub fn connect_to(self, pin: impl Peripheral<P = impl PeripheralOutput>) {
         crate::into_mapped_ref!(pin);
 
+        // FIXME: disconnect previous connection(s)
```

Comment:
Do we have an issue for this one? Sounds like a bug in waiting for a user tries to use the same pins for different drivers (maybe this isn't a common case, but let's track it at the very least)

Reply:
https://github.com/esp-rs/esp-hal/issues/2928